package-lock.json

```diff
@@ -5,6 +5,7 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "perun-web-apps",
```

Comment:
not sure what this is doing here

Reply:
it appeared after I used "npm install". Since this file is generated, it shouldn't be an issue.